Resources/public/js/models/ez-contentmodel.js

```diff
@@ -173,16 +175,20 @@ YUI.add('ez-contentmodel', function (Y) {
          */
         relations: function (type, fieldDefinitionIdentifier) {
             var relations,
-                fieldDefFilter = (typeof fieldDefinitionIdentifier !== "undefined");
+                fieldDefFilter = (typeof fieldDefinitionIdentifier !== "undefined"),
+                relationType = (typeof type !== "undefined");
```

Comment:
bad naming, `relationType` is a boolean telling whether a type was given or not, should be named `typeFilter` to be consistent with the previous line.
Also, if type is not provided, you can directly the content of the `relations` attribute.